pages/countries.js

```diff
@@ -118,9 +118,9 @@ const StyledRegionLink = styled.a`
 `
 
 const RegionLink = ({ href, label }) => (
-  <Box px={3} py={4}>
+  <Box px={[1,3]} py={[2,4]}>
```

Comment:
Vertical spacing `py` can go one step higher here.

Reply:
I tried increasing the `py` but it was not going perfectly with the UI so I didn't change it.